packages/expect/src/__tests__/extend.test.ts

```diff
@@ -15,21 +15,22 @@ const jestExpect = require('../');
 expect.addSnapshotSerializer(alignedAnsiStyleSerializer);
 
 jestExpect.extend({
-  toBeDivisibleBy(actual, expected) {
+  toBeDivisibleBy(actual: number, expected: number) {
```

Comment:
I suppose in a real matcher actual would always be `unknown`, but here for simplicity we can ignore that

Reply:
If i use `unknown`, TS does not allow the `%`operation on the next line (i guess i should add a `if typeof` before)

Reply:
Yes I understand, it was more of a loud thought. Could introduce a type check so that this would be more of a 'proper' matcher implementation. In a test where it's just supposed to be some example matcher, I'd be fine with keeping it as is. Your call :upside_down_face:

Reply:
Ok, i didn't understand your comment. You mean something like that:
```typescript
  toBeDivisibleBy(actual: unknown, expected: number) {
    let pass = false;
    if (typeof actual === 'number') {
      pass = actual % expected === 0;
    }
    const message = pass
      ? () => `expected ${actual} not to be divisible by ${expected}`
      : () => `expected ${actual} to be divisible by ${expected}`;

    return {message, pass};
  },
```